tests/console/consoletest_finish.pm

```diff
@@ -30,6 +30,7 @@ sub run() {
             if ( check_var( "DESKTOP", "gnome" ) ) {
                 send_key "esc";
                 unless ( get_var("LIVETEST") ) {
+                    send_key "ctrl"; # show gnome screen lock in sle 11
```

Comment:
In sle 11 previous esc hides gnome screen lock window, ctrl will show gnome screen lock window